dvc/output/gs.py

```diff
@@ -35,6 +35,12 @@ def changed(self):
 
         return False
 
+    def changed_cache(self):
```

Comment:
I've introduced OutputBase class in #1470 , so we can now define `changed_cache` in OutputBase once. See `OutputBase.changed()` as an example.